### ISSUE TITLE
Sample systemd service file does not work

### ISSUE THREAD

**Issue by kaysond** [airsonic/airsonic#1077]
I'm running Ubuntu 18.04 with 
```
administrator@fserver:~$ systemctl --version
systemd 237
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN -PCRE2 default-hierarchy=hybrid
```

contrib/airsonic.service doesn't run out of the box because `SystemCallFilter` should be delimited with spaces. Not sure how it behaves on other systemd versions. Happy to open a PR

**Comment by jooola** [comment]
Thanks for the report !

**Comment by jvoisin** [comment]
Fixed by #1078
